app/assets/stylesheets/_shame.scss

```diff
@@ -9,13 +9,6 @@
   color: $error-colour;
 }
 
-.ui-filter {
```

Comment:
is this not still used in `app/views/case_workers/admin/allocations/_allocation.html.haml:48`

Reply:
The class `.ui-filter` is used on the above view and elsewhere, however all views have inline styles `style: 'width:100%'`.
Inline styles will be actioned in ticket CBO-992

Reply:
where is that inline style? I cannot see it in in the above view or the `filter_tasks` partial. I ask because this is a particularly convoluted part of the app (filtering of allocation list options).

would need manually check the allocation function on this branch to feel more confident. 

can this wait til after cutover?

Reply:
`app/views/case_workers/admin/allocations/_allocation.html.haml` renders a partial (`app/views/case_workers/admin/allocations/_filter_tasks.html.haml`) that contains a select field. The select field has inline css `style: 'width: 100%'`.

Yes, this can wait until the cutover is complete.